node/src/components/linear_chain/event.rs

```diff
@@ -44,6 +46,9 @@ impl<I: Display> Display for Event<I> {
             Event::NewLinearChainBlock { block, .. } => {
                 write!(f, "linear chain new block: {}", block.hash())
             }
+            Event::KnownLinearChainBlock(block) => {
```

Comment:
It looks like that variant isn't constructed anywhere anymore. Should it be removed?